haiku-apps/beaccessible/beaccessible-1.0.0.recipe

```diff
@@ -0,0 +1,46 @@
+SUMMARY="An Access-like front-end to SQLite"
+DESCRIPTION="BeAccessible, an Access-like front-end to the SQLite database."
+HOMEPAGE="https://github.com/HaikuArchives/BeAccessible"
+COPYRIGHT="2004 randar"
+LICENSE="MIT"
+REVISION="1"
+COMMIT="3d9f8ff6b106375680422ab155bcb1b0815c9884"
+SOURCE_URI="https://github.com/HaikuArchives/BeAccessible/archive/$COMMIT.tar.gz"
+CHECKSUM_SHA256="097a5b5556353336bbb2772879c94d57e5f2e952cb31d1d5dcf62b358ee30701"
+SOURCE_DIR="BeAccessible-$COMMIT"
+
+ARCHITECTURES="x86_gcc2 !x86 !x86_64"
+
+PROVIDES="
+	beaccessible = $portVersion
+	app:BeAccessible = $portVersion
+	"
+REQUIRES="
+	haiku
+	lib:libexpat
+	lib:libsqlite3
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	devel:libexpat
+	devel:libsqlite3
+	"
+BUILD_PREREQUIRES="
+	makefile_engine
+	cmd:g++
+	cmd:make
+	"
+
+BUILD()
+{
+	cd src
+	make $jobArgs OBJ_DIR=objects
```

Comment:
```diff
-	cd src
-	make $jobArgs OBJ_DIR=objects
+	make -C src OBJ_DIR=objects $jobArgs
```

Reply:
I had just learned "make -C" and wondered why the recipes I saw didn't use it...